mst/integrations/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Integrations for Managed Service for TimescaleDB
+excerpt: Integrate Managed Service for TimescaleDB with other services.
+product: mst
```

Comment:
```suggestion
product: mst
products: [mst]
```
We're in a bit of a transitional period here, where we're moving from `product` (singular, string) to `products` (plural, array). This change is on the Cloud-centric launch branch, to support the new product icons. In the meantime, we should use both fields on the main branch to avoid problems when merging the launch branch in.